util/src/rlp/untrusted_rlp.rs

```diff
@@ -55,6 +55,18 @@ pub struct PayloadInfo {
 	pub value_len: usize,
 }
 
+fn calculate_payload_info(header_bytes: &[u8], len_of_len: &usize) -> Result<PayloadInfo, DecoderError> {
```

Comment:
reference to `usize` is the same size as `usize` itself -- better to pass-by-value